decidim-api/README.md

```diff
@@ -0,0 +1,28 @@
+# Decidim::Api
+Short description and motivation.
+
+## Usage
+How to use my plugin.
```

Comment:
I've created the exact same file we have in the other engines, mostly for symmetry. We should probably think about that to put there in the future.